### PR TITLE
Move efiling-frontend into frontend directory

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

This PR includes the following proposed change(s):

- Move efiling-frontend into frontend directory

## Type of change

- [x] Refactoring / Documentation

## How Has This Been Tested?

Local

## Review

@bashbang

### CODE REVIEW

**Comment by akroon3r** [comment]
paambaati is not happy:

error Command "coverage" not found.

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+exclude_patterns:
```
Comment (akroon3r) [inline .codeclimate.yml:1]: thank you 🚢

**Comment by akroon3r** [comment]
One conflict

**Comment by sdevalapurkar-bcgov** [comment]
Code climate issues should not be a blocker for this PR
Can be resolved in a different branch as the diff is already v large

**Pull request merged**